Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml

```diff
@@ -30,12 +30,14 @@ ParserQuery: |
     ASimAuthenticationAADServicePrincipalSignInLogs   (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationAADServicePrincipalSignInLogs'      in (DisabledParsers) )),
     ASimAuthenticationAWSCloudTrail (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationAWSCloudTrail'      in (DisabledParsers) )),
     ASimAuthenticationBarracudaWAF  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationBarracudaWAF' in (DisabledParsers) )),
+    ASimAuthenticationBarracudaCEF  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationBarracudaCEF' in (DisabledParsers) )),
```

Comment:
can you please attach the tester result for each parser? https://learn.microsoft.com/en-us/azure/sentinel/normalization-develop-parsers#test-parsers

Reply:
okay